ddl/ddl_api.go

```diff
@@ -1815,8 +1790,9 @@ func handleTableOptions(options []*ast.TableOption, tbInfo *model.TableInfo) err
 	if tbInfo.PreSplitRegions > tbInfo.ShardRowIDBits {
 		tbInfo.PreSplitRegions = tbInfo.ShardRowIDBits
 	}
-
-	return nil
+	var err error
+	tbInfo.Charset, tbInfo.Collate, err = getCharsetAndCollateInTableOption(0, options)
+	return err
```

Comment:
The enclosing method `handleTableOptions` is called in `buildTableInfoWithCheck`, which has already called `getCharsetAndCollateInTableOption` at line 1312 at this point.  Is the `getCharsetAndCollateInTableOption` here redundant?

Reply:
+1

Reply:
You're right, I just forgot to remove those codes.

Thanks!